### DISCUSSION TITLE
请问springboot怎么集成它？

### DISCUSSION THREAD

**Discussion by zhanghengdashuaibi** [reata/sqllineage#314] [General]
请问springboot怎么集成它？

**Comment by reata** [comment]
sqllineage是一个Python Library，非要和Spring Boot集成，两种办法：

1. 单独启动一个Python的微服务，通过接口来调用。
2. https://github.com/reata/sqllineage4j 是一个Java实现，目前只有表级别的血缘解析，并且暂时还没有发到Maven Central（预计年底之前会release）。可以先自行打包。

另外未来关于使用方面的问题，请提到Discussions板块，语言不限。Issues主要用来记录Bug和新Feature，因为contributor有来自国外的，Issues尽量使用英文交流。

**Reply by lzzcj** [reply]
我就是通过方式1实现的调用

**Reply by AyangOVOY** [reply]
请问您是怎么实现的呀
